hardhat.config.js

```diff
@@ -1,7 +1,27 @@
-require('@nomicfoundation/hardhat-chai-matchers');
-require('@nomiclabs/hardhat-ethers');
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
```

Comment:
Since it's an older file
```suggestion
 * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
```

Reply:
Thanks for the comment @Nana-EC! Should I update all the headers with `Copyright (C) 2022-2024 Hedera Hashgraph, LLC` or just this file?